model_demos/cv_demos/vovnet/pytorch_vovnet_ese_99b_hf_timm.py

```diff
@@ -1,12 +1,14 @@
 # VoVNet Model V2
 
+import os
 import urllib
 
 import pybuda
 import requests
 import timm
 import torch
 from PIL import Image
+from pybuda._C.backend_api import BackendDevice
```

Comment:
Remove -- not called anywhere

Reply:
done here : https://github.com/tenstorrent/tt-buda-demos/pull/28/commits/69b603789bba4b4ffef8f892a4f4e11e94b44cb2